tests/unit/expected_test_results/additives/fr-emulsifiant-chlorure.json

```diff
@@ -14,6 +12,8 @@
    "lc" : "fr",
    "minerals_tags" : [],
    "nucleotides_tags" : [],
-   "other_nutritional_substances_tags" : [],
+   "other_nutritional_substances_tags" : [
+      "en:choline-chloride"
```

Comment:
here it was used an additive

Reply:
understood. It was duplicates between additives.txt and other_nutritional_substances.txt, the solution was to keep it only in the latter one as there is already a block "en: Choline chloride" there. If Choline chloride is a synonym or a child of E1001, it could be reorganized in the taxonomy. But it might be out of the scope of this PR for now.